apps/web-desmos/src/screens/proposal_details/components/overview/index.tsx

```diff
@@ -90,10 +107,19 @@ const Overview: FC<{ className?: string; overview: OverviewType }> = ({ classNam
           </>
         );
       }
-    });
 
+      if (type.includes('MsgUpdateParams')) {
+        extraDetails = (
+          <>
+            {changes.map((change) => (
+              <UpdateParams changes={change} className="accordion" />
```

Comment:
We are missing a `key` prop here (could use the list index, not sure if `change.type` is unique)